Plan/common/src/main/resources/assets/plan/web/js/xmlhttprequests.js

```diff
@@ -59,10 +59,12 @@ function refreshingJsonRequest(address, callback, tabID, skipOldData) {
  * Make an XMLHttpRequest for JSON data.
  * @param address Address to request from
  * @param callback function with (json, error) parameters to call after the request.
+ * @param postBody If supplied, use POST method instead of GET.
  */
-function jsonRequest(address, callback) {
+function jsonRequest(address, callback, postBody) {
```

Comment:
I recommend making a new method instead of adding arguments to this method to avoid cluttering the method body

Reply:
I wanted to avoid duplicating the stuff inside of the onreadystatechange but I guess i can extract that body into a new method